crates/sncast/tests/e2e/verify/voyager.rs

```diff
@@ -0,0 +1,488 @@
+use crate::helpers::constants::{
+    ACCOUNT_FILE_PATH, CONTRACTS_DIR, MAP_CONTRACT_ADDRESS_SEPOLIA, MAP_CONTRACT_CLASS_HASH_SEPOLIA,
+};
+use crate::helpers::fixtures::copy_directory_to_tempdir;
+use crate::helpers::runner::runner;
+use indoc::formatdoc;
+use serde_json::json;
+use shared::test_utils::output_assert::assert_stderr_contains;
+use starknet_types_core::felt::Felt;
+use wiremock::matchers::{body_partial_json, method, path};
+use wiremock::{Mock, MockServer, ResponseTemplate};
+
+#[tokio::test]
+async fn test_happy_case_contract_address() {
+    let contract_path = copy_directory_to_tempdir(CONTRACTS_DIR.to_string() + "/map");
+
+    let mock_server = MockServer::start().await;
+    let rpc_response = json!({
+        "id": 1,
+        "jsonrpc": "2.0",
+        "result": MAP_CONTRACT_CLASS_HASH_SEPOLIA
+    });
+
+    let mock_rpc = MockServer::start().await;
+    let mock_rpc_uri = mock_rpc.uri().clone();
+
+    // Only mock the getClassHashAt call that voyager actually makes
+    Mock::given(method("POST"))
+        .and(body_partial_json(
+            json!({"method": "starknet_getClassHashAt"}),
+        ))
+        .respond_with(ResponseTemplate::new(200).set_body_json(rpc_response))
+        .expect(1)
+        .mount(&mock_rpc)
+        .await;
+
+    let job_id = "2b206064-ffee-4955-8a86-1ff3b854416a";
+    let class_hash = Felt::from_hex(MAP_CONTRACT_CLASS_HASH_SEPOLIA).expect("Invalid class hash");
+
+    Mock::given(method("POST"))
+        .and(path(format!("class-verify/{class_hash:#068x}")))
```

Comment:
Let's change it in tests as well
```suggestion
        .and(path(format!("class-verify/{class_hash:#064x}")))
```